common/jenkins-slaves/nodejs12/README.md

```diff
@@ -0,0 +1,54 @@
+# Node.js 12 - Jenkins Slave
+
+## Introduction
+
+This slave is used to build [Node.js v12.x](https://nodejs.org/dist/latest-v12.x/docs/api/) based projects, through `npm`, `npx` and `yarn`. The [`jenkins-slave-base`](https://github.com/opendevstack/ods-core/tree/master/jenkins/slave-base) is used as base image.
+
+The image is built in the global `cd` project and is named `jenkins-slave-nodejs12`.
+It can be referenced in a `Jenkinsfile` with `cd/jenkins-slave-nodejs12`.
+
+```groovy
+odsPipeline(
+    image: "${dockerRegistry}/cd/jenkins-slave-nodejs12",
+)
+```
+
+## Features
+
+1. Nodes.js v12.x
+2. npm / npx v6.x
+3. yarn v1.X
+4. Nexus configuration
+5. HTTP Proxy awareness
+
+## Known limitations
+
+n/a
```

Comment:
I would also add something like ``This slave is missing packages needed for running headless mode-based end-to-end and unit tests for frontend development, e.g. cypress, Karma, Protractor, etc. Please use the nodejs*-angular slave for this purpose.``

Reply:
Since these are not included, I do not see any limitation here 🤷‍♂️

Reply:
yeah, you are right. I just thought a hint would be nice. The concept of missing packages on an OS level that are just needed by browser-based testing frameworks is not straightforward. It might be pretty frustrating to figure out, what went wrong in your FE project.

Reply:
I know what you mean and I guess that's when release notes comes into play.

Reply:
As a frontend developer I would look into this ``README.md`` in order to judge, if I can use it for my frontend project or not. Just from reading the current version I would not sense any sign of "danger". To point it out: Using this slave in one of the QSs fe-angular, fe-ionic, fe-vue, and e2e-cypress will break unit or e2e test functionality.

Since we have two types of nodejs slaves we should also point out somewhere the difference between them and the reasoning for that.